pkg/crc/services/dns/dns_darwin.go

```diff
@@ -23,18 +23,18 @@ type resolverFileValues struct {
 
 func runPostStartForOS(serviceConfig services.ServicePostStartConfig, result *services.ServicePostStartResult) (services.ServicePostStartResult, error) {
 	// Write resolver file
-	success, err := createResolverFile(serviceConfig.HostIP, filepath.Join("/", "etc", "resolver", serviceConfig.BundleMetadata.ClusterInfo.BaseDomain))
+	success, err := createResolverFile(serviceConfig.IP, filepath.Join("/", "etc", "resolver", serviceConfig.BundleMetadata.ClusterInfo.BaseDomain))
 	// we pass the result and error on
 	result.Success = success
 	return *result, err
 }
 
-func createResolverFile(hostIP string, path string) (bool, error) {
+func createResolverFile(InstanceIP string, path string) (bool, error) {
```

Comment:
maybe this has to be unnamed to which IP this is, like `ipAddress`, since the calling function in that case uses `instanceIP` or `hostIP`.